src/components/buttons/button-embed-video.jsx

```diff
@@ -46,10 +46,12 @@ class ButtonEmbedVideo extends React.Component {
 		} else {
 			return (
 				<button
+					arial-label={AlloyEditor.Strings.video}
```

Comment:
s/arial/aria/

I see what you did there...

![Screen Shot 2019-05-27 at 16 01 51](https://user-images.githubusercontent.com/7074/58424907-d34e0200-8098-11e9-8e54-f1630321ab8f.png)

Reply:
Nice good catch